src/web/publiser/publiser.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "HTML: Ekstra! Publiser nettsiden din"
+level: 2
+language: nb-NO
+---
+# Publiser nettsiden din på Internett{ .activity }
+
+Nå har du laget ditt eget nettsted. Du ønsker vel å vise det frem, gjør du ikke?
+
+Hvis du bare kopierer adressen til websiden fra nettleseren din, og deretter sender den til noen, ville de ikke se den. Det er fordi denne adressen beskriver et sted på datamaskinen din, og vennene dine ikke har tilgang til den. Selv om hadde hatt det, hva om de ønsket å se på den når datamaskinen din er skrudd av?
+
+Husker du servere fra den første økten? Servere er datamaskiner som alltid er på og koblet til Internett, og de er satt opp slik at folk kan besøke nettsteder som lever på disse datamaskinene.
+
+For å gjøre dette vil vi bruke dropbox eller google drive. Hvis du ikke har en egen konto kan du spørre om å få bruke mamma eller pappa sin.
```

Comment:
Kanskje vi kan finne en måte å gjøre dette uten konto? Paste i jsbin? Gist + rawgit?

http://stackoverflow.com/questions/10717169/view-rendered-output-of-a-gist
http://jsbin.com/lasaxewaze